app/Providers/RouteServiceProvider.php

```diff
@@ -49,7 +50,7 @@ public function boot()
     protected function configureRateLimiting()
     {
         RateLimiter::for('api', function (Request $request) {
-            return Limit::perMinute(60);
+            return Limit::perMinute(6);
```

Comment:
@driesvints do we want to agressively limit the whole API this much or just these routes we're not expecting to receive much traffic? I know we don't expect any of these routes to be hit hard, but thinking about other routes we may wish to add later down the line.

Reply:
Happy to be more specific here. Just let me know what you want 👍

Reply:
@driesvints what do you think?

Reply:
Let's keep this for now 👍